docs/user_guide/50_satellite_overview.md

```diff
@@ -0,0 +1,39 @@
+# Satellite Lab Overview
+
+## What is a Satellite Lab?
+Satellite labs receive patient samples from reference labs that don't have the necessary equipment to handle certain tests. The results are then sent back to the reference lab through BLIS.
```

Comment:
The use of "sent" here is a little confusing - maybe instead, something like: "The reference lab uses BLIS to store the results, which can also be accessed by the satellite labs"?